docs/en/latest/FAQ.md

```diff
@@ -371,3 +371,34 @@ You may have to install OPENSSL in your system and/or pass OPENSSL_DIR or OPENSS
 Example: luarocks install luasec OPENSSL_DIR=/usr/local
 make: *** [deps] Error 1
 ```
+
+## How to access APISIX Dashboard through APISIX proxy
+
+1. Keep APISIX and Admin API using different ports (or disable Admin API), for example, do the following configuration in `conf/config.yaml`.
```

Comment:
I think `APISIX` and `Admin API` do not peer to peer, should be "Keep the proxy port and Admin API port of APISIX different".

Reply:
updated.